File: 1-easy/problems/2194-cells-in-a-range-on-an-excel-sheet/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def cellsInRange(self, s):
        """
        :type s: str
        :rtype: List[str]
        """
        return [chr(x)+chr(y) for x in range(ord(s[0]), ord(s[3])+1) for y in range(ord(s[1]), ord(s[4])+1)]

File: 1-easy/problems/2194-cells-in-a-range-on-an-excel-sheet/solution.ts
function cellsInRange(s: any): boolean | number | string | any {
  """
  :type s: str
  :rtype: List[str]
  """
  return [chr(x)+chr(y) for x in Array.from({length: ord(s[0]}, (_, i) => i), ord(s[3])+1) for y in Array.from({length: ord(s[1]}, (_, i) => i), ord(s[4])+1)];
}

export { cellsInRange };
